.github/workflows/copy-updated-schema.yml

```diff
@@ -0,0 +1,49 @@
+name: Sync Schema Update to Website
+
+on:
+  push:
+    branches:
+      - master
+      - automate-spec-update-to-website
+
+    paths:
+     # Monitor changes in this v3 file
+      - schemas/3.0.0-without-$id.json
+
+jobs:
+  sync-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout spec-json-schema repo
+        uses: actions/checkout@v3
+        
+      - name: Checkout website repo
+        uses: actions/checkout@v3
+        with:
+          repository: asyncapi/website
+          token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
+          path: ./website
+
+      - name: Copy version 3 Schema File
+        run: |
+          cp schemas/3.0.0-without-\$id.json website/config/3.0.0.json
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

Comment:
for security reasons please never use versions in case of actions that are not coming from GH

reference to sha of the release like so: https://github.com/asyncapi/spec-json-schemas/blob/master/.github/workflows/if-nodejs-version-bump.yml#L52

Reply:
Out of curiosity, what are the security implication of not using the `sha` of the release?

Reply:
https://www.linkedin.com/posts/lukasz-gornicki-a621914_supplychainattack-github-cve202530066-activity-7308457205618352129-QW3H?utm_source=share&utm_medium=member_desktop&rcm=ACoAAAC88aoBtJebjfqQZ-iTK5AuS7Ceb8yntWc